dspace-api/src/main/resources/Messages.properties

```diff
@@ -3912,7 +3917,9 @@ jsp.search.results.cris.classcerif = CERIF Semantics Classification
 jsp.layout.hku.label.propertiesdefinition.rendering.pointer.path	=	URL Path for resource
 help.jdyna.message.rendering.pointer.path	=	URL is in the form of //authority/path where 'authority' is the context path of the webapp while the 'path' is the location of the resource within the authority, in this case the expression in the url path will be evaluated (e.g. insert here a path like cris/uuid/${displayObject.uuid})
 jsp.layout.table.cris.admin-list.actions = Actions
-jsp.layout.do.detail.name = Research Object
+jsp.layout.do.detail.name = {0} {1}
```

Comment:
this is the default way to build the cris page title, cris type label <space> cris object name but you can override it for specific cris type so to translate it in multiple languages if needed as the cris type label cannot be translated from the UI